pkg/helper/template/generate/template.yaml

```diff
@@ -0,0 +1,7 @@
+name: bunnyshell - {{ .Basename }}
+description: Bunnyshell template for {{ .Basename }}
+tags: []
+icons: []
+stack:
+  packages: []
```

Comment:
I would also add a dummy package here, with the name/value struct, for the example, and also to make it valid - it's invalid like this (with empty packages).